public/src/reimbursement-policy.md

```diff
@@ -2,107 +2,121 @@
 
 ### Venues and associated costs
 
-- Reimbursement requests can be made for venues used for a Speedcubing Canada event.
+- Reimbursement requests are allowed to be made for venues used for Speedcubing Canada Supported Events.
 - Expenses covered by this policy include:
   - Venue rental fees
   - Non-refundable damage deposits
   - Special event liability insurance
   - Wired or wireless internet access
-  - Audio-visual
+  - Audio-visual services or equipment rental
   - Risers
   - Tables
   - Chairs
   - Belt barriers or rope and stanchions
 
 ### Operating expenses
 
-- Reimbursement requests can be made for operating expenses of a Speedcubing Canada event.
+- Reimbursement requests are allowed to be made for operating expenses of Speedcubing Canada Supported Events.
 - Expenses covered by this policy include:
   - World Cube Association Competition Dues
-  - Stripe Fees
 
 ### Equipment
 
-- Reimbursement requests can be made for equipment used for a Speedcubing Canada event.
+- Reimbursement requests are allowed to be made for equipment used for Speedcubing Canada Supported Events.
 
-#### Type 1
+#### Type 1: timing equipment
 
-- Equipment used during scrambling and solving in an official World Cube Association competition.
 - Expenses covered by this policy include:
   - StackMat timers
   - StackMat mats
   - Tournament displays
   - Tournament display data cords
+  - Tournament display power adaptors
   - Tournament display stands
   - Stopwatches
-  - Batteries
-  - O-rings
+  - O-rings and other button protectors
+
+#### Type 2: non-timing equipment
+
+- Expenses covered by this policy include:
   - Cube covers
-  - Stationary (e.g. pens, markers)
-  - Tri-folds
-  - Laptops and tablets (only if necessary for scrambling stations)
-  - Harmonica holders (only if necessary for 4x4x4 Blindfolded, 5x5x5 Blindfolded or 3x3x3 Multi-Blind)
-  - Cardstock and paper (e.g. for scorecards, FMC, 3x3x3 Blindfolded, 4x4x4 Blindfolded, 5x5x5 Blindfolded or 3x3x3 Multi-Blind)
+  - Scrambling station blockers (e.g. tri-folds, pipe and drape)
+  - Blind blockers and harmonica holders
+  - Laptops, tablets and required accessories (e.g. dongles, power adaptors)
+  - Extension cords and power bars
+  - Batteries
 
-#### Type 2
+#### Type 3: printing and paper
 
-- Equipment not used during scrambling and solving in an official World Cube Association competition, but still necessary for the successful operation of a Speedcubing Canada event.
 - Expenses covered by this policy include:
-  - Name badges
-  - Lanyards
-  - Notepads
-  - Table cloths
-  - Professional printing (e.g. programs, banners, table runners, stickers)
-  - Cardstock and paper (e.g. for name tags, schedules or certificates)
+  - Cardstock and paper
+  - Professional printing (e.g. banners, table cloths, table runners, print programmes, signs, stickers, name badges)
```

Comment:
I think 'program' is more common in Canadian English
[Canadian Oxford Dictionary](https://www.oxfordreference.com/view/10.1093/acref/9780195418163.001.0001/acref-9780195418163?avail_01=free&avail_02=unlocked&btog=chap&hide=true&jumpTo=program&page=2762&pageSize=20&skipEditions=true&sort=titlesort&source=%2F10.1093%2Facref%2F9780195418163.001.0001%2Facref-9780195418163)

Reply:
fixed